third-party/fltk/Makefile

```diff
@@ -32,12 +32,12 @@ $(FLTK_BUILD_DIR):
 	@mv fltk-$(FLTK_VERSION) $(FLTK_BUILD_DIR)
 
 clean:
-	cd $(FLTK_BUILD_DIR) && $(MAKE) clean
-	rm -f build.$(CHPL_HOST_PLATFORM) install.$(CHPL_HOST_PLATFORM)
+	-cd $(FLTK_BUILD_DIR) && $(MAKE) clean
+	-rm -f build.$(CHPL_HOST_PLATFORM) install.$(CHPL_HOST_PLATFORM)
 
 cleanall:
-	cd $(FLTK_BUILD_DIR) && $(MAKE) cleanall
-	rm -f build.$(CHPL_HOST_PLATFORM) install.$(CHPL_HOST_PLATFORM)
+	-cd $(FLTK_BUILD_DIR) && $(MAKE) cleanall
+	-rm -f build.$(CHPL_HOST_PLATFORM) install.$(CHPL_HOST_PLATFORM)
 
 clobber:
 	-rm -rf $(FLTK_BUILD_DIR) $(FLTK_INSTALL_DIR) *~
```

Comment:
This isn't related to your changes, but I'm uneasy about the `rm -rf *~`, why is this needed?